main.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_acm_certificate" "certificate" {
+  domain_name               = var.domain_name
+  subject_alternative_names = var.alternate_domain_names
+  validation_method         = "DNS"
+
+  tags = local.merged_tags
+
+  options {
+    certificate_transparency_logging_preference = var.enable_certificate_transparency_log ? "ENABLED" : "DISABLED"
+  }
+}
+
+resource "aws_route53_record" "validation" {
+  count = length([var.domain_name, var.alternate_domain_names])
+
+  zone_id = var.route53_zone_id
+  name    = aws_acm_certificate.certificate.domain_validation_options[count.index].resource_record_name
+  type    = aws_acm_certificate.certificate.domain_validation_options[count.index].resource_record_type
+  ttl     = 60
```

Comment:
Should this be hardcoded?

Reply:
The `ttl` value in this case?

Yeah, I believe that makes sense.

Oversimplified: the lower this value is, the quicker AWS ACM will pick up the change.